### DISCUSSION TITLE
Hidden Traceable

### DISCUSSION THREAD

**Discussion by SonoSooS** [ppy/osu#18114] [Ideas]
Basically the complement of #13109.

Enabling Hidden + Traceable would show approach circles briefly before fading out, with the hit object body being never shown.

Currently Hidden + Traceable when enabled, completely hides everything but slider edges, which is basically almost like some permanent blinds mod, which is not really what I want.

Considering how #13109 was implemented, I think the easiest would be to just add this as an option into the Hidden mod. Although that would be slightly counter-intuitive, considering Traceable already exists. I also don't see how Hidden + Traceable could detect eachother in a sane way, so I don't know how I would approach implementing this.

**Comment by mk56-spn** [comment]
https://user-images.githubusercontent.com/74463310/181801077-37f4c942-d516-446f-8262-d6f940634574.mp4

implemented as a subsetting for traceable since it seemed like the sanest way, will PR if this implementation is workable
